api-gateway/src/main/resources/static/scripts/vet-details/vet-details.controller.js

```diff
@@ -3,6 +3,7 @@
 angular.module('vetDetails')
     .controller('VetDetailsController', ['$http', '$stateParams', function ($http, $stateParams) {
         var self = this;
+        //var vetId = $stateParams.vetId || 0;
```

Comment:
Let's not forget to take out the commented code we have, just to make it clean.

Reply:
Will also be needed to have a vetId variable declare for user storie (694) on Joira